apps/site/pages/en/learn/asynchronous-work/discover-promises-in-nodejs.md

```diff
@@ -381,7 +381,7 @@ console.log('Synchronous task executed');
 
 - Use `queueMicrotask()` for tasks that need to run immediately after the current script and before any I/O or timer callbacks, typically for Promise resolutions.
 - Use `process.nextTick()` for tasks that should execute before any I/O events, often useful for deferring operations or handling errors synchronously.
-- Use `setImmediate()` for tasks that should run after I/O events but before timers.
+  – Use `setImmediate()` for tasks that should run after the current event loop's I/O events.
```

Comment:
The issue wasn't the spacing, it's the dash character. It should match the lines before it.